modules/ninja/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "homepage": "https://ninja-build.org/",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```

Comment:
Would you be willing to become a module maintainer?

Reply:
At present, I'd prefer to avoid becoming the Bazel module maintainer for projects that I don't have an upstream commit bit for.